src/platform/tgatexture.cpp

```diff
@@ -93,7 +93,7 @@ detail::TextureImpl* Load( File& F )
         Succ = F.Read( Buf, InSize );
         if( Succ )
         {
-            detail::TextureImpl::ConvertRGBtoRGBA( ( uint8_t* )( void* )Buf.c_str(), ImgSize, &impl->mData.at( 0 ) );
```

Comment:
Don't know, tbh. suppressing is possible probably. Gonna try with "validating" the input width/height (<= 2^15)